### ISSUE TITLE
Initial reactions to using extension

### ISSUE THREAD

**Issue by dpriedel** [neoclide/coc-python#17]
I now have this extension installed and running.

These comments are not directed to the coc-python extension but rather the Microsoft Python Language Server which provides the functionality behind it.

It seems the Microsoft Python LanguageServer does not do asynchronous linting -- only on file save or if manually invoked.

Based on comments in the issues section of the Microsoft Python LanguageServer github repo, the latest version takes a long long time and lots of memory to do its initial source scan.  While it is doing this, it is not availble for code completion.  This seems to be new behavior -- there are a lot of complaints. On my system, for a small (150 lines) Python program which uses pandas and matplotlib, it takes about 2 minutes to become usable.  When it does become ready code completion is quite good.

So, currently, I am running this extension with linting diabled so I am only using code completion.  

I am using ALE for linting.

Bottom line...this extension seems to be working OK but the underlying language server has some issues.

**Comment by chemzqm** [comment]
> Microsoft Python LanguageServer does not do asynchronous linting

It requries saved file to work.

>  takes a long long time and lots of memory to do its initial source scan.

It depends on python modules you have installed on your system.

I've changed coc-python to use jedi by default, same as vscode-python.

**Comment by chemzqm** [comment]
MSPL is still beta, you can try upgrade language server to see if it can work for you now.
